sphinxcontrib/confluencebuilder/rest.py

```diff
@@ -17,32 +19,82 @@
 from .exceptions import ConfluenceProxyPermissionError
 from .exceptions import ConfluenceSeraphAuthenticationFailedUrlError
 from .exceptions import ConfluenceTimeoutError
+from .exceptions import ConfluenceSSLError
+from .exceptions import ConfluenceCertificateError
 from .std.confluence import API_REST_BIND_PATH
 
+class SSLAdapter(HTTPAdapter):
```

Comment:
(optional) While I know PEP 8 (and maybe Google's Python guidelines) promote "CapWords" style, I'd prefer the convention `SslAdapter`. If you want to leave it as is or if @tonybaloney prefers the "CapWords" style, that's fine by me.